Removed time parsing workaround for old Openstack version.

varikin opened the pull request:
Since Kilo isn't supported anymore, we don't need to try parsing the
time with a timezone and fallback to without the timezone anymore.

Commit:
Removed time parsing workaround for old Openstack version.

Since Kilo isn't supported anymore, we don't need to try parsing the
time with a timezone and fallback to without the timezone anymore.

varikin committed:
Lookup owner of AMI in AllowLaunch operation. (#1010)

If the AMI is baked with a different account that the one being used to
deploy, that owner account needs to be used to update the permissions on
that AMI.

tomaslin committed:
titus - don't set a job group sequence if the requested type is batch.

tomaslin committed:
Merge pull request #1026 from tomaslin/dont-set-sequence

Titus - don't set job sequence if type is batch

lwander committed:
(consul & google) enable/disable instance (#1017)

varikin committed:
Added the Openstack Swift Provider. (#1027)

Currently just reads a file saved in a Swift object.

lwander committed:
new banner (#982)

lwander committed:
(kubernetes) Translate objects before loading them into the cache (#1020)

Commit:
provider/openstack: Allow server groups to be created without loadbalancer. (#1028)

closes https://github.com/spinnaker/spinnaker/issues/1092

derekolk commented on clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/provider/agent/OpenstackServerGroupCachingAgent.groovy:
```diff
@@ -220,6 +219,16 @@ class OpenstackServerGroupCachingAgent extends AbstractOpenstackCachingAgent imp
       .build()
   }
 
+  long getCreationTime(Stack stack) {
```
My two cents is that I would create a parseZonedDateTime(String time, ZonedDateTime defaultTime) inside of DateUtils in case other operations or places need this.

varikin replied:
I thought about after I deleted DateUtils and found that the logic left in place got a little uglier.

milismsft committed:
Bug fix - failed to select vnet in arbitrary resource group (#1029)

When creating an azure load balancer and server group, the selected virtual network can be from a different resource group then the one corresponding to the application.
